### DISCUSSION TITLE
Should Azd integrate with Landing Zone Accelerators?

### DISCUSSION THREAD

**Discussion by TWolversonReply** [Azure/azure-dev#2710] [Ideas]
The Awesome-Azd templates generally implement an application-level architecture; there's a frontend, a backend, and some services that make it all hang together.

In the enterprise, we need to go a level above that. We call the process of specifying and configuring an environment so that an organization can work in the cloud at scale a [Landing Zone](https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/landing-zone/), and Microsoft provides [Landing Zone Accelerators](https://techcommunity.microsoft.com/t5/apps-on-azure-blog/announcing-landing-zone-accelerator-for-azure-container-apps/ba-p/3843989) to help with automating landing zone provision.

Does Azd aim to work in the same scope as Landing Zone Accelerators? Should Accelerators be Azd-based (or -enabled) going forward?

**Comment by savannahostrowski** [comment]
Many of the Landing Zone accelerators are azd-compatible today. We are working closely with the LZA team to make all accelerators azd-compatible.

**Reply by TWolversonReply** [reply]
Great - glad to hear it. It seems like the only one that isn't, from the ones I've looked at, is the one that I was most interested in, the Integration Services accelerator.
